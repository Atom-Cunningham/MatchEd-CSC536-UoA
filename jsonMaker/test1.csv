color, number, size, name
yellow, 4, large, harry
red, 3, small, bill
green, 5000000000, nosize, jenny