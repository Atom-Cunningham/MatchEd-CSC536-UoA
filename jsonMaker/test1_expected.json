[
	{
		" name": " harry",
		" number": " 4",
		" size": " large",
		"color": "yellow"
	},
	{
		" name": " bill",
		" number": " 3",
		" size": " small",
		"color": "red"
	},
	{
		" name": " jenny",
		" number": " 5000000000",
		" size": " nosize",
		"color": "green"
	}
]